modules/installation-special-config-kmod.adoc

```diff
@@ -0,0 +1,304 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_user_infra/installing-aws-user-infra.adoc
+
+[id="installation-user-infra-generate-k8s-manifest-ignition-rhcos_{context}"]
+= Adding kernel modules to nodes
+
+For most common hardware, the Linux kernel includes the device driver
+modules needed to use that hardware when the computer starts up. For
+some hardware, however, modules are not available in Linux. So you need
+to find a way to provide those modules to each host computer. This
+procedure describes how to do that for nodes in an {product-title} cluster.
+
+When a kernel module is first deployed following these instructions,
+the module is made available for the current kernel. If a new kernel
+is installed, the kmods-via-containers software will rebuild and deploy
+the module so a compatible version of that module is available with the
+new kernel.
+
+The way that this feature is able to keep the module up to date on each
+node is by:
+
+* Adding a systemd service to each node that starts at boot time to detect
+if a new kernel has been installed and
+* If a new kernel is detected, the
+service rebuilds the module and installs it to the kernel
+
+For information on the software needed for this procedure, see the
```

Comment:
We're not able to reference GitHub: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites

Reply:
I'll talk to Eric about this tomorrow. Without this content there is no procedure. The content doesn't exist anywhere else yet.